### ISSUE TITLE
Pan that turns to pinch doesn't work

### ISSUE THREAD

**Issue by davidnx** [uber-web/mjolnir.js#57]
Verified with `react-map-gl 5.2.3` in its default configuration on Android Chrome 80.0.3987.132.

Repro steps:
* Start a `"pan"` gesture (i.e. using one finger, start to drag the map around). Don't release the finger
* While still holding the first finger, use another finger to initiate a pinch action.

**Expectation:** map stays "attached" to both finger touch points, and pans and zooms as a physical object would.
**Actual:** The gesture halts, the map stops moving (doesn't pan nor pinch) until one of the fingers is released.

I have been able to work around this by changing [RECOGNIZER_COMPATIBLE_MAP ](https://github.com/uber-web/mjolnir.js/blob/cebeb585c3e842b79118d67dc9354af854b48a31/src/constants.js#L40) to also allow a pan recognizer with pinch, i.e. added the line below:

```js
  pinch: ['pan'],
```

In my use case I don't care about rotation, but I expect something like this would have to be done for that to work properly as well.

**Comment by Pessimistress** [comment]
@davidnx Thanks. Would you open a PR to fix this?

**Comment by Pessimistress** [comment]
Published 2.4.1

**Comment by alex-dolbik** [comment]
I still cannot  pan and zoom at the same time. Updated packages.

![image](https://user-images.githubusercontent.com/10628158/80687278-bb65d600-8ad2-11ea-9ef5-e3842213defe.png)

**Comment by davidnx** [comment]
@Pessimistress thanks, the fix worked great and I've been able to remove the work-around I had prior. Verified on `react-map-gl@5.2.5`.

@alex-dolbik I'd suggest filing a separate issue with additional details on your issue to help folks here repro and diagnose. The fix provided here solved my issue, so I am closing this one. I am not a maintainer of this project though, and will defer to @Pessimistress if she has other suggestions.
